sspl_test/plans/sanity.pln

```diff
@@ -1,4 +1,15 @@
 alerts.node.test_node_disk_actuator
+alerts.realstor.test_real_stor_controller_sensor
```

Comment:
Good to see these back in sanity, we need this badly to keep out sanity really sane!